core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -5630,32 +5658,49 @@ components:
           description: migration attempt number for this workspace
         created:
           type: string
+          format: date-time
           description: when the migration was scheduled
         started:
           type: string
+          format: date-time
           description: when the system started the migration
         updated:
           type: string
+          format: date-time
           description: when the system last updated the migration
         finished:
           type: string
+          format: date-time
           description: when the system finished executing the migration
         outcome:
           $ref: '#/components/schemas/WorkspaceMigrationOutcome'
     WorkspaceMigrationOutcome:
       description: Represents the result of a workspace migration attempt
-      type: object
-      required:
-        - type
+      oneOf:
+        - $ref: '#/components/schemas/WorkspaceMigrationSuccess'
+        - $ref: '#/components/schemas/WorkspaceMigrationFailure'
       properties:
-        type:
-          type: string
-          enum:
-            - Success
-            - Failure
-        message:
+        failure:
           description: Supplementary failure information
           type: string
+    WorkspaceMigrationSuccess:
+        type: string
+        enum:
+          - success
+    WorkspaceMigrationFailure:
+        type: object
+        required:
+          - failure
+        properties:
+          failure:
+            description: Supplementary failure information
+            type: string
+    BatchMigrateRequestBody:
+      description: List of workspace names to migrate
+      type: array
+      items:
+        type: string
+        pattern: "^{namespace}/{name}$"
```

Comment:
not sure what the correct format is (or if we have one). At least in the swagger page it makes things clear